src/api/db/migrate/20201013143657_add_bio_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddBioToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :bio, :string
```

Comment:
It would be nice to have an empty string as a default value so we don't have to check for `nil` in the code.
```suggestion
    add_column :users, :bio, :string
    change_column_default :users, :bio, ''
```

We cannot set the default value directly in the `add_column` statement because our MariaDB version will block writes to a table when adding a column with a default value. See [`Adding a column with a default value` in strong_migrations' documentation](https://github.com/ankane/strong_migrations/tree/1e133224579fe708fd01324504cd4e8fc3f29e71#adding-a-column-with-a-default-value).

Reply:
I doubted between "bio" and "biography," so second one is ok for me too.

However, setting an empty string as default value doesn't prevent us to do some check, since we don't want to display an empty `<p></p>`. I wonder what's the benefit. :thinking:

Reply:
It's one less check to do if we don't have to deal with `nil`.

Reply:
is it some best practice? Otherwise I don't see the point to add `change_column_default` and a backfill migration if I'm going to use `.blank?` anyway.

Reply:
Working with `nil` is for sure more error-prone. Even its creator said it was [an error to invent this concept](https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare/). He calls `nil` (or `null`) his billion dollar mistake.

We don't need to have the default value, it's just to avoid potential errors later.

Reply:
Thanks for your explanation, @dmarcoux :+1:

Reply:
@dmarcoux updated replacing "bio" by "biography" and setting empty string as default value.